manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -4735,6 +4716,51 @@ subjects:
     name: istiod-service-account
     namespace: istio-system
 ---
+# Source: base/templates/clusterrole.yaml
```

Comment:
Any reason for moving those here ? Generally namespace-level roles are in the regular template.